cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -126,7 +126,7 @@ function(ADD_ARROW_LIB LIB_NAME)
 
   set(RUNTIME_INSTALL_DIR bin)
 
-  if (ARROW_BUILD_SHARED)
+  if (${ARROW_BUILD_SHARED})
```

Comment:
Hmm... wow. What is the difference? :-)

Reply:
I was testing out something and left this artifact. Will remove

Reply:
The difference is that the old version is the variable/variable-name and the new one is the content of the variable. In the case of `if` this is irrelevant but matters at many more places.